platform/viewer/src/lib/DownloadViewportEngine.js

```diff
@@ -0,0 +1,137 @@
+const DOWNLOAD_ELEMENT_ID = 'download-element';
+const PREVIEW_ELEMENT_ID = 'preview-element';
+
+class DownloadViewportEngine {
```

Comment:
What is meant by "Engine"?

Reply:
Engine is the name I gave to where I connect to cornerstone, cornerstone tools and others libs like the one to generate blobs.